src/github.com/mozilla-services/pushgo/simplepush/etcd_locator.go

```diff
@@ -45,6 +45,11 @@ type EtcdLocatorConf struct {
 	// will be considered valid. Defaults to "10s".
 	RefreshInterval string `toml:"refresh_interval" env:"refresh_interval"`
 
+	// StartDelay is the amount of time to wait after registering the host with
+	// etcd. This should be 1-2 times the refresh interval to ensure the locator
+	// has a complete view of the cluster.
+	StartDelay string `toml:"start_delay" env:"start_delay"`
```

Comment:
We could make this a multiplier, too; I just used an explicit duration to match `CloseDelay`.